docs/contributing.rst

```diff
@@ -67,14 +67,17 @@ The following tools are there to help you:
 Integration
 ~~~~~~~~~~~
 
-First, install `Go`_ 1.5 and start Boulder_, an ACME CA server::
+First, install `Go`_ 1.5 (pick a value for GOPATH and put $GOPATH/bin in your
```

Comment:
Environment variables set in a shell script won't propagate up to the parent shell.  So the attempt to set GOPATH there isn't working.